contrib/podmanimage/README.md

```diff
@@ -13,7 +13,7 @@ using the latest Fedora and then Podman is installed into them:
   * quay.io/podman/stable - This image is built using the latest stable version of Podman in a Fedora based container.  Built with podman/stable/Dockerfile.
   * quay.io/podman/upstream - This image is built using the latest code found in this GitHub repository.  When someone creates a commit and pushes it, the image is created.  Due to that the image changes frequently and is not guaranteed to be stable.  Built with podmanimage/upstream/Dockerfile.
   * quay.io/podman/testing - This image is built using the latest version of Podman that is or was in updates testing for Fedora.  At times this may be the same as the stable image.  This container image will primarily be used by the development teams for verification testing when a new package is created.  Built with podmanimage/testing/Dockerfile.
-
+  * quay.io/podman/stable:version - This image is built 'by hand' using a Fedora based container.  An RPM is first pulled from the [Fedora Updates System](https://bodhi.fedoraproject.org/) and the image is built from there.  For more details, see the Containerfile used to build it, podmanimage/stablebyhand/Containerfile.podmanstable
```

Comment:
I suggest in replacing "by hand" with "manually" if that is the intention. I’m also not sure if we want to put the Containerfile into a `stablebyhand` directory or just `stable`/`manual`.

Reply:
But I like my coloquialism!  I've changed it to manually.  The Containerfile can live anywhere when being run, but I think you're referring to its location in the repo.  I've moved it to `podmanimage/stable/manual/Containerfile.  Will push a new variant shortly.

Reply:
Cool thank you :pray: